ssz-rs/src/bitlist.rs

```diff
@@ -87,7 +86,7 @@ impl<const N: usize> Bitlist<N> {
 
     fn pack_bits(&self) -> Result<Vec<u8>, MerkleizationError> {
         let mut data = vec![];
-        let _ = self.serialize_with_length(&mut data, false)?;
+        let _ = self.serialize_with_length(&mut data, false);
```

Comment:
we want to keep this right?

Reply:
Adressed in https://github.com/ralexstokes/ssz-rs/pull/25/commits/438a003498f6a5be024b30a8137c8673eb05895a.